### PR TITLE
UI styling

### PR DESCRIPTION
**Pull request by fi-ya**
This merge includes the complete stories for the trello ticket [UI style changes](https://trello.com/c/3EV66vWe/18-ui-style-changes)

- [x] Update numbering 1-9

- [x] Display which player won

- [x] Display which player made the move and have a new line underneath

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -17,42 +17,70 @@ class Board
...
   end
 
   def position_taken?(index)
-    board[index] == 'X' || board[index] == 'O'
+    board[index - 1] == 'X' || board[index - 1] == 'O'
   end
 
   def valid_move?(index)
-    !position_taken?(index) && index.between?(0, 8) ? true : false
+    !position_taken?(index) && index.between?(1, 9) ? true : false
```
Comment (Himalee) [inline lib/board.rb:51]: I think this can be refactored to just: 
```
!position_taken?(index) && index.between?(1, 9)

File: lib/board.rb [hunk 2]
```diff
@@ -17,42 +17,70 @@ class Board
...
   end
 
-  def generate_board
-    " #{@board[0]} | #{@board[1]} | #{@board[2]} \n" \
-      "-----------\n" \
-      " #{@board[3]} | #{@board[4]} | #{@board[5]} \n" \
-      "-----------\n" \
-      " #{@board[6]} | #{@board[7]} | #{@board[8]} \n"
+  def get_player_mark(board)
+    if board.count('X') == board.count('O')
```
Comment (Himalee) [inline lib/board.rb:25]: Could the X's and O's be refactored out into 2 constants? That way we are only defining the values of 'X' and 'O' in one place. Then everywhere we need to use the X or O values, we would call the constant instead. This way, if we ever wanted to change the values, we would only need to update it in one place.

File: lib/board.rb [hunk 3]
```diff
@@ -17,42 +17,70 @@ class Board
   ].freeze
 
   def initialize(message)
-    @board = %w[0 1 2 3 4 5 6 7 8]
+    @board = %w[1 2 3 4 5 6 7 8 9]
```
Comment (Himalee) [inline lib/board.rb:20]: Nice!

File: lib/board.rb (hunk 2 above)
Comment (Himalee) [reply lib/board.rb:25]: More about constants here: https://www.tutorialspoint.com/ruby/ruby_variables.htm

**Comment by Himalee** [comment]
This is looking good! Just a note that the PR also introduces new logic that determines whether the game has reached a winning state, which goes beyond the ticket to implement UI styling updates. Was that intentional to deliver the UI styling changes ticket?

File: lib/board.rb [hunk 4]
```diff
@@ -3,7 +3,7 @@
 require_relative 'message'
 
 class Board
-  attr_accessor :board
+  attr_accessor :board, :players_move
 
   WINNING_MOVES = [
```
[the remark this answers is not in the document]
Comment (fi-ya) [reply lib/board.rb:8]: It keeps track of the players input. I needed to have access to it to pass into messages for the output string that repeats what move the player has made.
Screenshot 2022-03-09 at 15 42 30

**Pull request merged**